src/interfaces.ts

```diff
@@ -126,11 +127,12 @@ export interface FormResults {
 }
 
 export interface FormData {
-  results: FormResults;
-  errorObj: {};
+  results: FormResults | undefined;
+  errorObj: ProblemDetail | undefined;
```

Comment:
Why can `results` and `errorObj` be undefined now?

Reply:
There won't be any data initially and the error will only be there if the API returns one.